src/tilepy/include/PointingTools.py

```diff
@@ -1607,7 +1617,134 @@ def ZenithAngleCut(prob, nside, time, minProbcut, maxZenith, observatory, minMoo
     return ObsBool, yprob
 
 
-def ComputeProbability2D(prob, highres, radecs, reducedNside, HRnside, minProbcut, time, observatory, maxZenith, FOV, tname, ipixlist, ipixlistHR, counter, dirName, useGreytime, plot):
+def GetSatelliteName(satellitename):
```

Comment:
please no hardcoded txt in the code... you can use this function to pre-launching tilepy, to get the coordinates/name. The name of the satellite should be in the config anyways.

Reply:
resolved